pkg/util/expectations/resource_version_expectation.go

```diff
@@ -37,7 +38,8 @@ func NewResourceVersionExpectation() ResourceVersionExpectation {
 
 type realResourceVersionExpectation struct {
 	sync.Mutex
-	objectVersions map[types.UID]string
+	objectVersions            map[types.UID]string
+	firstUnsatisfiedTimestamp time.Time
```

Comment:
`firstUnsatisfiedTimestamp` should not put in global realResourceVersionExpectation